survey_contact_generation/models/survey_question.py

```diff
@@ -18,11 +18,13 @@ class SurveyQuestion(models.Model):
     @api.depends("question_type")
     def _compute_allowed_field_ids(self):
         type_mapping = {
-            "textbox": ["char"],
+            "textbox": ["char", "text"],
             "free_text": ["text"],
-            "numerical_box": ["integer", "float"],
-            "date": ["date"],
-            "datetime": ["datetime"],
+            "numerical_box": ["integer", "float", "text", "char"],
+            "date": ["date", "text", "char"],
+            "datetime": ["datetime", "text", "char"],
+            "simple_choice": ["many2one", "text", "char"],
+            "multiple_choice": ["many2many", "text", "char"],
```

Comment:
When I try to use a multiple_choice question on a char field, i select in the survey more than one option but in the contact the field is filled with only one of the answers. Isn't this the same behaviour as the simple_choice?

What i was expecting is some kind of string concatenation between the answers selected. Doe this make any sense? Should be reconsider the multiple_choice questions possibilities?

Reply:
Good catch, it should return all the responses. Can you add the fix for it?

Reply:
Done, please review it.